core/src/main/java/nl/nn/adapterframework/scheduler/job/CheckReloadJob.java

```diff
@@ -49,45 +50,56 @@ public void execute(IbisManager ibisManager) {
 		List<String> configNames = new ArrayList<>();
 		List<String> configsToReload = new ArrayList<>();
 
-		FixedQuerySender qs = SpringUtils.createBean(getApplicationContext(), FixedQuerySender.class);
-		qs.setDatasourceName(getDataSource());
-		qs.setQuery("SELECT COUNT(*) FROM IBISCONFIG");
-		String booleanValueTrue = qs.getDbmsSupport().getBooleanValue(true);
-		String selectQuery = "SELECT VERSION FROM IBISCONFIG WHERE NAME=? AND ACTIVECONFIG = "+booleanValueTrue+" and AUTORELOAD = "+booleanValueTrue;
-		try {
-			qs.configure();
-			qs.open();
-			try (Connection conn = qs.getConnection(); PreparedStatement stmt = conn.prepareStatement(selectQuery)) {
-				for (Configuration configuration : ibisManager.getConfigurations()) {
-					String configName = configuration.getName();
-					configNames.add(configName);
-					if ("DatabaseClassLoader".equals(configuration.getClassLoaderType())) {
-						stmt.setString(1, configName);
-						try (ResultSet rs = stmt.executeQuery()) {
-							if (rs.next()) {
-								String ibisConfigVersion = rs.getString(1);
-								String configVersion = configuration.getVersion(); //DatabaseClassLoader configurations always have a version
-								if(StringUtils.isEmpty(configVersion) && configuration.getClassLoader() != null) { //If config hasn't loaded yet, don't skip it!
-									log.warn(getLogPrefix()+"skipping autoreload for configuration ["+configName+"] unable to determine [configuration.version]");
-								}
-								else if (!StringUtils.equalsIgnoreCase(ibisConfigVersion, configVersion)) {
-									log.info(getLogPrefix()+"configuration ["+configName+"] with version ["+configVersion+"] will be reloaded with new version ["+ibisConfigVersion+"]");
-									configsToReload.add(configName);
+		if(!atLeastOneConfigrationHasDBClassLoader) {
+			for (Configuration configuration : ibisManager.getConfigurations()) {
+				if("DatabaseClassLoader".equals(configuration.getClassLoaderType())) {
+					atLeastOneConfigrationHasDBClassLoader = true;
+					break;
+				}
+			}
+		}
```

Comment:
Could you add a notification that it hasn't found any database things and thus skips it?